### ISSUE TITLE
[Users] Implement undo user delete feature

### ISSUE THREAD

**Issue by 0xnmn** [coding-blocks/gondor#135]
Once https://github.com/coding-blocks/gondor/issues/134 is resolved, we need a way to undo the delete by the admins. 

- [ ] add a `userEnable` mutation to undo the delete.
- [ ] allowed only for admins.
- [ ] add an argument to the users query type named 'includeDisabled' based on which you include the disabled entities in the results too. By default it can be false. Even if it is true, the disabled entities shall only be included if the user is admin.
- [ ] in the users query type, the default order should be based on the deleted_at, and should also be used as a fallback even if another column is provided via args to sort based on.

**Comment by jugshaurya** [comment]
@hereisnaman, on it!

**Comment by Ayush-Mahajan** [comment]
Is this issue still open?

**Comment by 0xnmn** [comment]
> Is this issue still open?

Yes

**Comment by jugshaurya** [comment]
> Is this issue still open?

Yes working on it. Just waiting for #134 to be merged.

**Comment by Ayush-Mahajan** [comment]
So can I start working on it or should I wait for another issue to open up. I am new to BOSS so i dont have much experience with how exactly does the scoring and all works @hereisnaman

**Comment by jugshaurya** [comment]
> So can I start working on it or should I wait for another issue to open up. I am new to BOSS so i dont have much experience with how exactly does the scoring and all works @hereisnaman 
> 

@Ayush-Mahajan, welcome to open source. There are other issues as well besides this. But you are more than welcome to work on this issue as well. But since this issue depends upon #134 issue of which i have submitted a PR, just waiting for its testing. You can start by testing my PR as well. #138

**Comment by Ayush-Mahajan** [comment]
@jugshaurya thanks. I tried looking for other issues but i think all of them are taken

**Comment by jugshaurya** [comment]
> @jugshaurya thanks. I tried looking for other issues but i think all of them are taken
> 

As of now nothing is taken you can work on any issue and submit a PR. If they like it it will get merged. By the out of 6 issues in this repo 4 are not taken by anyone or no one has worked in last 5 days. So you can work there.also try other repos like codingblocks boss repo. Good luck

**Comment by Ayush-Mahajan** [comment]
ohh thanks
